analysis/utils/plot_rates.R

```diff
@@ -0,0 +1,39 @@
+## ###########################################################
+
+##  This script:
+## - Contains a general function that is used for visualising the 
+##   mortality rates
+
+## linda.nab@thedatalab.com - 20220325
+## ###########################################################
+
+# Load libraries & functions ---
+library(dplyr)
+library(ggplot2)
+library(lubridate)
+
+## This function contains the common elements of a plot
```

Comment:
you could add one line explaining the input arguments for your function.
they are qute self explanorty but something along the lines:

df, a dataframe with variables describing x and y
y, character of the variable name for the y axis ...